### PR TITLE
API to expose ugc grace period

### LINKED ISSUE #4990: Sync uncommitted GC client run time to the linked address tokens
Getting a physical address token is valid for 6 hours (LinkAddressTime). make sure the GC is not configured to run less than 6 hours so it won't delete the addresses that haven't been linked.

### PR DESCRIPTION
**Pull request by itaiad200**
closes #4990

### CODE REVIEW

File: pkg/api/controller.go [hunk 1]
```diff
@@ -3651,6 +3652,19 @@ func (c *Controller) GetLakeFSVersion(w http.ResponseWriter, r *http.Request) {
...
+func (c *Controller) GetGarbageCollectionConfig(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	_, err := auth.GetUser(ctx)
+	if err != nil {
+		writeError(w, r, http.StatusUnauthorized, ErrAuthenticatingRequest)
+		return
+	}
+
+	writeResponse(w, r, http.StatusOK, GarbageCollectionConfig{
+		GracePeriod: aws.Int(int(ref.LinkAddressTime)),
```
Comment (N-o-Z) [inline pkg/api/controller.go:3664]: Can you add lakeFS version as well

**Review by N-o-Z — CHANGES_REQUESTED** [review]
One comment and missing test

File: pkg/api/controller.go (hunk 1 above)
Comment (itaiad200) [reply pkg/api/controller.go:3664]: You have `/config/version` endpoint for that.

**Review by N-o-Z — APPROVED** [review]
LGTM

**Pull request merged**